lib/features/mailbox/data/repository/mailbox_repository_impl.dart

```diff
@@ -48,51 +48,39 @@ class MailboxRepositoryImpl extends MailboxRepository {
       return MailboxResponse(mailboxes: response.first, state: response.last);
     });
 
-    yield localMailboxResponse;
-
-    if (localMailboxResponse.hasData()) {
-      bool hasMoreChanges = true;
-      State? sinceState = localMailboxResponse.state!;
```

Comment:
IMO, because we don't use it anymore, please remove line 46 `stateDataSource.getState(accountId, session.username, StateType.mailbox)` also

Reply:
No, we still use it for the first `yield local to UI`. To save the old state to memory for `getChanges`